### DISCUSSION TITLE
Murray with custom CSS

### DISCUSSION THREAD

**Discussion by NikosAlexandris** [executablebooks/meta#995] [Q&A]
Hi all

I am trying to make the https://sphinx-themes.org/sample-sites/murray/ theme to get to work with https://github.com/patrickmineault/codebook/blob/main/docs/_static/tufte.css (and https://github.com/edwardtufte/et-book) to work in _light_ mode, or else, have all auto/dark/light options available.

In a similar way, as in https://github.com/patrickmineault/codebook, I have installed Jupyter-Book, the Murray theme and placed the custom CSS rules (plus the ET fonts) in the `_static` subdirectory. After building the HTML artefact, I get

![2023-04-02T13:15:05,813107899+03:00](https://user-images.githubusercontent.com/7046639/229346823-3ca44600-23b3-4773-9db6-33e1cc376369.png)

Removing the custom CSS (i.e. changing the `.css` filename's extension to something else), I get, as epxected I guess

![2023-04-02T13:14:12,697709487+03:00](https://user-images.githubusercontent.com/7046639/229346852-29cef003-c7b6-4cdb-919b-7a920a6dc4f1.png)

Is there an easy way to default to light mode, using the custom CSS rules ?  Or, turn on the toggle dark/light mode button?

**Comment by NikosAlexandris** [comment]
Pages I am also consulting : 

- https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/light-dark.html
- https://sphinx-book-theme.readthedocs.io/en/stable/sections/header.html#add-components-to-the-header-navbar

**Comment by ZiaCreatesIdeas** [comment]
I have a similar question. Hopefully you will get an answer.
